Dockerfile

```diff
@@ -50,6 +50,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-
     hipify-clang \
     half \
     libssl-dev \
+    libtbb-dev \
```

Comment:
This can be removed since its in the install_prereqs.

Reply:
Done